src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -71,6 +73,7 @@ private function findPlaces(Definition $definition, Marking $marking = null)
     {
         $places = array();
 
+        /** @var Place $place */
```

Comment:
I reverted all theses inline `@var` because:

* We almost never use them in Symfony
* It's useless (I know what you are going to say, no need to ;) )